### ISSUE TITLE
ValueError: invalid literal for int() with base 10: '2]\n[image:1 align:right'

### ISSUE THREAD

**Issue by bellini666** [django-wiki/django-wiki#1052]
Found this error in my installation after someone added this article:

```
Some Text.

[article_list depth:2]

[image:1 align:right]
````

Removing the `image` fixed the issue.

You can see the full stack from this error here: https://sentry.zerosoft.com.br/share/issue/7d13acc7b58f4d88a2c5549717b050ef/

edit. As you can see in the sentry link, I'm using `0.6b2` because I'm also using django 3.0

**Comment by benjaoming** [comment]
Seems like a problem in the macro regex.. instead of matching to the first `]`, it goes too far.

**Comment by slinkymanbyday** [comment]
not tested but this change may fix it.

```
MACRO_RE = r"((?i)\[(?P<macro>\w+)(?P<kwargs>\s\w+\:.*?)*\])"

KWARG_RE = re.compile(
    r"\s*(?P<arg>\w+)(:(?P<value>([^\'\]]+|%s)))?" % re_sq_short, re.IGNORECASE
)
```

**Comment by oscarmcm** [comment]
Closing this as further checks confirm that this issue may have been solved in the 6.0 release.

https://github.com/django-wiki/django-wiki/commit/6c801b2b271e16fbd17dcd0b4be857ea9c79713a#diff-f7e39b0527d5f5d2669304d30d47a3a7df2ae650946f4b76a1f3cbf5918afd5aL12
